src/contracts/DecentralandVesting.ts

```diff
@@ -19,32 +19,32 @@ export class DecentralandVesting extends Contract {
   }
 
   async duration() {
-    const bigNumber: BigNumber = await this.sendCall('duration')
+    const bigNumber: BigNumber = await this.sendCallByType('duration')
     return bigNumber.toNumber()
   }
 
   async cliff() {
-    const bigNumber: BigNumber = await this.sendCall('cliff')
+    const bigNumber: BigNumber = await this.sendCallByType('cliff')
     return bigNumber.toNumber()
   }
 
   async vestedAmount() {
-    const bigNumber: BigNumber = await this.sendCall('vestedAmount')
+    const bigNumber: BigNumber = await this.sendCallByType('vestedAmount')
     return eth.utils.fromWei(bigNumber.toNumber())
   }
 
   async releasableAmount() {
-    const bigNumber: BigNumber = await this.sendCall('releasableAmount')
+    const bigNumber: BigNumber = await this.sendCallByType('releasableAmount')
     return eth.utils.fromWei(bigNumber.toNumber())
   }
 
   async released() {
-    const bigNumber: BigNumber = await this.sendCall('released')
+    const bigNumber: BigNumber = await this.sendCallByType('released')
     return eth.utils.fromWei(bigNumber.toNumber())
   }
 
   async start() {
-    const bigNumber: BigNumber = await this.sendCall('start')
+    const bigNumber: BigNumber = await this.sendCallByType('start')
```

Comment:
this callByType doesn't work. Type is missing. What do we do when no arguments are necessary?